domains/anveshraman.json

```
@@ -5,5 +5,13 @@
   },
   "records": {
     "CNAME": "developeranveshraman.github.io"
```

you forgot the comma
```suggestion
    "CNAME": "developeranveshraman.github.io",
```